packages/shared/lib/db/migrations/20231213124728_drop_sync_data_records_indexes.cjs

```diff
@@ -0,0 +1,17 @@
+exports.config = { transaction: false };
+
+exports.up = function(knex) {
+    return knex.schema
+        .raw('DROP INDEX CONCURRENTLY created_at_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_data_hash_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_sync_job_id_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_external_is_deleted_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_external_deleted_at_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_deletes_data_hash_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_deletes_sync_job_id_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_deletes_external_is_deleted_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_data_records_deletes_external_deleted_at_index')
+        .raw('DROP INDEX CONCURRENTLY _nango_sync_jobs_deleted_index');
+};
+
+exports.down = function(knex) { };
```

Comment:
I choose not to revert the index drop so in case of migration rollback, we don't try to recreate those big indexes that could have a disastrous effect on performance